.github/workflows/update_dependabot_bundler_pr.yml

```diff
@@ -0,0 +1,67 @@
+# This workflow is based on https://github.com/dependabot/dependabot-actions-workflow
```

Comment:
Are there any changes to this workflow from the examples? I don't see anything related to the modules check.